cleaned_data/modsbyPID/rftaart:6.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mods xmlns="http://www.loc.gov/mods/v3"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:xlink="http://www.w3.org/1999/xlink"
+      xmlns:xs="http://www.w3.org/2001/XMLSchema"
+      xsi:schemaLocation="http://www.loc.gov/mods/v3 http://www.loc.gov/standards/mods/v3/mods-3-5.xsd">
+   <identifier type="local">RFTA_Braddock_006</identifier>
+   <identifier type="pid">rftaart:6</identifier>
+   <titleInfo>
+      <title>Stacey Adam and Chris Szaton evacuate</title>
+   </titleInfo>
+   <originInfo>
+      <dateIssued>2021</dateIssued>
+      <dateIssued encoding="edtf" keyDate="yes">2021</dateIssued>
+      <publisher/>
+      <place>
+         <placeTerm valueURI="http://id.loc.gov/authorities/names/n79109786">Knoxville (Tenn.)</placeTerm>
+      </place>
+   </originInfo>
+   <abstract>Stacey Adam and Chris Szaton packed bags and then worked all day cleaning leaves from gutters and lawn, showering their house with hoses—both the roof and grass around the house. At 6 p.m. someone came to the house and told them they had to leave now! Stacey mentioned her sense of empathy for her neighbors. Even though she and Chris didn’t lose their house, the fire was a life-altering experience to live through.</abstract>
+   <name authority="naf"
+         valueURI="http://id.loc.gov/authorities/names/n2002022941">
+      <namePart>Braddock, Paige</namePart>
+      <role>
+         <roleTerm authority="marcrelator"
+                   valueURI="http://id.loc.gov/vocabulary/relators/art">Artist</roleTerm>
+      </role>
+   </name>
+   <name>
+      <namePart>Adam, Stacey</namePart>
+      <role>
+         <roleTerm authority="marcrelator"
+                   valueURI="http://id.loc.gov/vocabulary/relators/ive">Interviewee</roleTerm>
+      </role>
+   </name>
+   <name>
+      <namePart>Szaton, Chris</namePart>
+      <role>
+         <roleTerm authority="marcrelator"
+                   valueURI="http://id.loc.gov/vocabulary/relators/ive">Interviewee</roleTerm>
+      </role>
+   </name>
+   <physicalDescription>
+      <form authority="aat" valueURI="http://vocab.getty.edu/aat/300033973">drawings (visual works)</form>
+   </physicalDescription>
+   <subject authority="lcsh"
+            valueURI="http://id.loc.gov/authorities/subjects/sh85146723">
+      <topic>Wildfires</topic>
+   </subject>
+   <subject authority="lcsh"
+            valueURI="http://id.loc.gov/vocabulary/graphicMaterials/tgm000665">
+      <topic>Automobiles</topic>
+   </subject>
+   <typeOfResource>text</typeOfResource>
+   <relatedItem displayLabel="Project" type="host">
+      <titleInfo>
+         <title>Rising from the Ashes: The Chimney Tops 2 Wildfires in Memory and Art.</title>
+      </titleInfo>
+   </relatedItem>
+   <relatedItem type="references">
+      <location>
+         <url>Waiting on interview transcript</url>
```

Comment:
@logantillman - We'll need to update these to actual links eventually, but this item is still unavailable as we don't have a captions file/transcript. For now you can pull out the location/url value and just leave the PID in the relatedItem parent element to remind us that this value is needed.